src/engraving/libmscore/image.cpp

```diff
@@ -536,7 +543,7 @@ void Image::layout()
     setPos(0.0, 0.0);
     if (imageType == ImageType::SVG && !svgDoc) {
         if (_storeItem) {
-            svgDoc = new QSvgRenderer(_storeItem->buffer());
+            svgDoc = new SvgRenderer(_storeItem->buffer());
```

Comment:
doesn't it make sense to just allocate him once?

Reply:
I think yes, needs refactoring for this class (by way, there is a memory leak too).
But I would not like to mix different tasks in one PR